README.md

```diff
@@ -1 +1,87 @@
 # VTEX Pages editor
+
+The Pages editor it's platform to dynamically edit a VTEX Store, making possible to select editable components and change its configurations adding or removing content in a simple and straightforward away.
```

Comment:
This is redundant: "...simple and straightforward away." remove the word **simple**